operations/finance.md

```diff
@@ -373,42 +373,39 @@ Returns all accounting items of the enterprise that were consumed \(posted\) or
 
 #### Accounting item subtype
 
-##### Revenue subtypes
-
-* `CancellationFee`
-* `Rebate`
-* `Deposit`
-* `ExchangeRateDifference`
-* `CustomItem`
-* `Surcharge`
-* `SurchargeDiscount`
-* `SpaceOrder`
-* `ProductOrder`
-* `Other`
-
-##### Payment subtypes
-
-* `CreditCard`
-* `Invoice`
-* `Cash`
-* `Unspecified`
-* `BadDebts`
-* `WireTransfer`
-* `ExchangeRateDifference`
-* `ExchangeRoundingDifference`
-* `BankCharges`
-* `Cheque`
-* `Other`
+* Revenue subtypes
+  * `CancellationFee`
+  * `Rebate`
+  * `Deposit`
+  * `ExchangeRateDifference`
+  * `CustomItem`
+  * `Surcharge`
+  * `SurchargeDiscount`
+  * `SpaceOrder`
+  * `ProductOrder`
+  * `Other`
+
```

Comment:
This blank line kind of breaks it, you can check it looks a bit weird when rendered here on github, if you remove it, it should be fine.